indigo_app/static/stylesheets/_tasks-view.scss

```diff
@@ -0,0 +1,39 @@
+// helper classes
+.task-list-card-full-height {
+  height: 100%;
+  max-height: 70vh;
+}
+
+.task-list-card-radius {
+  border-radius: 4px;
+}
+
+.task-list-card-scroll {
+  max-height: 100%;
+  overflow: scroll;
+}
+
+.task-list-card-current-task {
+  border: 1px solid $primary;
+  box-shadow: 0px 0px 3px 2px rgba(0,0,0,0.15);
+}
+
+.task-list-card-approval-task {
+  background-color: #f8f8f8;
+  border: 1px solid #dedede;
+}
+
+.approval-heading-span {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.task-list-card-heading {
+  text-decoration: underline;
+  color: black;
+}
+
+.task-list-card-approved-column {
```

Comment:
I suggest we make these border decorators something like `task-border-<<task-state>>`, eg `task-border-pending_review`, `task-border-assigned`, etc. That way they're programmatic and can be used anywhere to add a border coloured based on the state.

Reply:
Done